types/module/configurator.go

```diff
@@ -31,9 +32,14 @@ type Configurator interface {
 	// will panic. If the ConsensusVersion bump does not introduce any store
 	// changes, then a no-op function must be registered here.
 	RegisterMigration(moduleName string, forVersion uint64, handler MigrationHandler) error
+
+	// Cdc defines the app-wide codec interface used for serialization and
+	// deserialization.
+	Cdc() codec.Marshaler
```

Comment:
@aaronc @robert-zaremba adding Cdc inside Configurator came up multiple times. Should we just go ahead and add it here?

Reply:
Yes but also consider this potential refactoring: https://github.com/regen-network/regen-ledger/issues/301#issuecomment-804179925